Samples/DocumentCreationTest/PYTHON/DocumentCreationTest.py

```diff
@@ -36,15 +63,192 @@ def main():
     "deserunt mollit anim id est laborum.")
 
     result = True
-    
+
```

Comment:
Would be nice if we could minimize the non-required whitespace changes (particularly for a language like python). There are some at the end of the doc too.
It's fine if you're deliberately cleaning stuff up, but the changes don't seem to be consistent